Reconstruction/doc/RecCalorimeter.md

```diff
@@ -3,51 +3,102 @@ RecCalorimeter package
+
+## 3. Noise addition
+
+Noise is added to all cells, not only those with signal. Hence a list of all cellIDs is needed and it is retrieved from the geometry. This is the only part which is geometry dependent and assumes tube geometry and phi-eta segmentation.
 
-Cell energy reconstruction from Geant4 energy deposits.
+ `NoiseCaloCellsTool`: Prepare random noise hits for each cell.
+ `MergeCaloHitsTool`: Merge signal with noise hits.
+ `NoiseCaloCellsTool`: Filter cell energy and accept only cells with energy above threshold.
 
-Input: Geant4 hits (EDM: CaloHitCollection)
 
-Output: Calorimeter cells (EDM: CaloHitCollection)
+# Reconstruction
 
-**Tools used:**
+Reconstruction creates clusters (`fcc::CaloCluster`) out of cells (`fcc::CaloHit`). Each cluster stores the information about its global position (x, y, z), energy and the relation to the cells it is composed of.
 
-* MergeCaloHitsTool: Merge Geant4 energy deposits with same cellID
-* CalibrateCaloHitsTool: Calibrate Geant4 energy to electromagnetic (EM) scale
-* NoiseCaloCellsTool: Prepare random noise hits for each cell
+## Sliding window algorithm
 
-  A list of all cellIDs are needed for the noise tool (we want to simulate noise in all cells, not only in thouse with signal). This is done in the initialize method of the  algorithm. This is the only part which is geometry dependent (assuming tube geometry and phi-eta segmentation).
+Sliding window algorithm is an algorithm used for the reconstruction of electrons and photons. The algorithm is the same one as used by the ATLAS experiment [link](https://cds.cern.ch/record/1099735?ln=de).
 
-* MergeCaloHitsTool: Merge signal with noise hits
-* NoiseCaloCellsTool: Filter cell energy and accept only cells with energy above threshold
+`CreateCaloClustersSlidingWindow` implementation assumes the detector with eta-phi segmentation.
 
-**Configurables:**
+> Note: If this segmentation was not used for the original simulation and the true deposits' positions are saved (`fcc::PositionedCaloHit`, it is possible to resegment using `RedoSegmentation` algorithm.
 
-* hits (input data)
-* cells (output data)
-* doCellCalibration (bool, calibrate energy to EM scale?)
-* addCellNoise (bool, add noise to cells?)
-* filterCellNoise (bool, save only cells with energy above threshold?)
-* readoutName (string, name of the detector readout)
-* activeVolumeName (string, name of active volumes - material name)
-* numVolumesRemove (unsigned, number of volumes with active material which are not readout - e.g. ECAL: LAr bath in cryostat)
-* activeFieldName (string, name of active layers for sampling calorimeter)
-* fieldNames (vector of string, name of the fields describing the segmented volume)
-* fieldValues (vector of int, values of the fields describing the segmented volume);
+### 1. Create calorimeter towers.
```

Comment:
it's just [above](https://github.com/HEP-FCC/FCCSW/pull/148/files#diff-441eef0619f2939a0e8856ca7587ee89R55)